### PR TITLE
feat: Added the Part 01 Tutorial in Hindi

### PR DESCRIPTION
**Pull request by darkmatter18**
Ref: #2774 

- Created a Hindi folder inside tutorial folder
- Translated Part 01 - The Basic Tools of Private Deep Learning in Hindi

Signed-off-by: Arkadip <in2arkadipb13@gmail.com>

### CODE REVIEW

File: examples/tutorials/hindi/Part 01 - The Basic Tools of Private Deep Learning.ipynb [hunk 1]
```diff
@@ -0,0 +1,965 @@
...
+    "\n",
+    "**2) उद्यमी अवसर** - समाज में समस्याओं की एक पूरी मेजबानी है जिसे डीप लर्निंग हल कर सकता है, लेकिन कई सबसे महत्वपूर्ण का पता नहीं लगाया गया है क्योंकि इसे लोगों के बारे में अविश्वसनीय रूप से संवेदनशील जानकारी तक पहुंच की आवश्यकता होगी (मानसिक या रिश्ते के मुद्दों के साथ लोगों की मदद करने के लिए डीप लर्निंग का उपयोग करने पर विचार करें) !)। इस प्रकार, निजी डीप लर्निंग सीखना आपके लिए नए स्टार्टअप अवसरों की एक पूरी मेजबानी को अनलॉक करता है जो पहले इन टूलसेट के बिना दूसरों के लिए उपलब्ध नहीं थे।\n",
+    "\n",
+    "**3) सोशल गुड** - डीप लर्निंग का उपयोग वास्तविक दुनिया में कई प्रकार की समस्याओं को हल करने के लिए किया जा सकता है, लेकिन डीप लर्निंग व्यक्तिगत जानकारी पर डीप लर्निंग लोगों के बारे में, लोगों के लिए है। दीप लर्निंग कैसे करना है डेटा पर सीखना आपके पास कैरियर या उद्यमशीलता के अवसर से अधिक का प्रतिनिधित्व नहीं करता है, यह लोगों के जीवन में कुछ सबसे व्यक्तिगत और महत्वपूर्ण समस्याओं को हल करने में मदद करने का अवसर है - और इसे पैमाने पर करने के लिए।\n",
+    "\n",
+    "## मुझे अतिरिक्त क्रेडिट कैसे मिलेगा?\n",
+    "\n",
+    "- Pysyft को Github पर लाइक करें! - [https://github.com/OpenMined/PySyft](https://github.com/OpenMined/PySyft)\n",
```
Comment (Yugandhartripathi) [inline examples/tutorials/hindi/Part 01 - The Basic Tools of Private Deep Learning.ipynb]: Like can be changed with star here for following through with proper terminology.

File: examples/tutorials/hindi/Part 01 - The Basic Tools of Private Deep Learning.ipynb [hunk 2]
```diff
@@ -0,0 +1,965 @@
...
+   ],
+   "source": [
+    "x_ptr"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "BOOM! अब बॉब(Bob) के दो टेनर्स हैं! मुझे विश्वास नहीं है? अपने लिए इसे देख लें!"
```
Comment (Yugandhartripathi) [inline examples/tutorials/hindi/Part 01 - The Basic Tools of Private Deep Learning.ipynb]: Correct tensor translation and change मुझे with मुझपे

File: examples/tutorials/hindi/Part 01 - The Basic Tools of Private Deep Learning.ipynb [hunk 3]
```diff
@@ -0,0 +1,965 @@
...
+   ],
+   "source": [
+    "bob._objects"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "अब कुछ नोटिस करो। जब हमने `x.send (bob)` को फोन किया तो उसने एक नई वस्तु लौटा दी जिसे हमने` x_ptr` कहा। यह एक टेंसर के लिए हमारा पहला  पॉइंटर  है। दसियों को इंगित करने वाले वास्तव में स्वयं डेटा नहीं रखते हैं। इसके बजाय, वे बस एक मशीन पर संग्रहीत एक डेटा (डेटा के साथ) के बारे में मेटाडेटा होते हैं। इन टेंसरों का उद्देश्य हमें एक सहज ज्ञान युक्त एपीआई देना है जो अन्य मशीन को इस टेंसर का उपयोग करके कार्यों की गणना करने के लिए बताए। आइए मेटाडेटा पर एक नज़र डालें जिसमें संकेत होते हैं।"
```
Comment (Yugandhartripathi) [inline examples/tutorials/hindi/Part 01 - The Basic Tools of Private Deep Learning.ipynb]: Spacing issue with x_ptr need to swap out space in backticks to the left. I think data outside parenthesis here(डेटा (डेटा के साथ)) should be tensor instead. What do you think?

Comment (Yugandhartripathi) [reply examples/tutorials/hindi/Part 01 - The Basic Tools of Private Deep Learning.ipynb]: Also in the last line here word pointers is replaced with संकेत need to fix that for it to make sense

File: examples/tutorials/hindi/Part 01 - The Basic Tools of Private Deep Learning.ipynb [hunk 4]
```diff
@@ -0,0 +1,965 @@
...
+    "x_ptr"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "उस मेटाडेटा (Metadata) की जाँच करें!\n",
+    "\n",
+    "संकेत करने के लिए विशिष्ट दो मुख्य विशेषताएं हैं:\n",
```
Comment (Yugandhartripathi) [inline examples/tutorials/hindi/Part 01 - The Basic Tools of Private Deep Learning.ipynb]: Again change संकेत with pointers here.  It seems the translator interprets pointers as in tips or signals when it falls in the last part of a sentence.

File: examples/tutorials/hindi/Part 01 - The Basic Tools of Private Deep Learning.ipynb [hunk 5]
```diff
@@ -0,0 +1,965 @@
...
+    "\n",
+    "- `x_ptr.location : bob`, स्थान, उस स्थान का संदर्भ जिसे सूचक (Pointer) इंगित कर रहा है\n",
+    "- `x_ptr.id_at_location : <random integer>`, वह आईडी जहां स्थान पर Tensor संग्रहीत होता है\n",
+    "\n",
+    "वे प्रारूप में मुद्रित होते हैं `<id_at_location>@<location>`\n",
+    "\n",
+    "अन्य अधिक सामान्य विशेषताएँ भी हैं:\n",
+    "- `x_ptr.id : <random integer>`, हमारे पॉइंटर Tensor की आईडी, इसे बेतरतीब ढंग (Randomly Allocated) से आवंटित किया गया था\n",
+    "- `x_ptr.owner : \"me\"`, वर्कर जो पॉइंटर (Pointer) टेनर का मालिक है, यहाँ उसका स्थानीय कर्मी है, जिसका नाम \"me\" है\n"
```
Comment (Yugandhartripathi) [inline examples/tutorials/hindi/Part 01 - The Basic Tools of Private Deep Learning.ipynb]: टेनर to Tensor

File: examples/tutorials/hindi/Part 01 - The Basic Tools of Private Deep Learning.ipynb [hunk 6]
```diff
@@ -0,0 +1,965 @@
...
+   ],
+   "source": [
+    "me == x_ptr.owner"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "और अंत में, जैसे हम एक Tensor पर .send () कॉल कर सकते हैं, हम इसे वापस पाने के लिए एक Tonsor के लिए पॉइंटर (Pointer) पर .get () कॉल कर सकते हैं !!!"
```
Comment (Yugandhartripathi) [inline examples/tutorials/hindi/Part 01 - The Basic Tools of Private Deep Learning.ipynb]: Tonsor to Tensor

File: examples/tutorials/hindi/Part 01 - The Basic Tools of Private Deep Learning.ipynb [hunk 7]
```diff
@@ -0,0 +1,965 @@
...
+   "source": [
+    "x_ptr.owner"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "आपको आश्चर्य हो सकता है कि स्थानीय कर्मचारी जो सूचक (Pointer) का मालिक है, वह भी एक VirtualWorker है, हालांकि हमने इसे नहीं बनाया है।\n",
+    "मजेदार तथ्य, ठीक उसी तरह जैसे बॉब (Bob) के लिए हमारे पास VirtualWorker ऑब्जेक्ट था, हम (डिफ़ॉल्ट रूप से) हमेशा हमारे लिए एक होते हैं। जब हम `hook = sy.TorchHook ()` कहते हैं, तो यह कार्यकर्ता स्वचालित रूप से बनाया जाता है और इसलिए आपको आमतौर पर इसे स्वयं बनाने की आवश्यकता नहीं है।"
```
Comment (Yugandhartripathi) [inline examples/tutorials/hindi/Part 01 - The Basic Tools of Private Deep Learning.ipynb]: "हम (डिफ़ॉल्ट रूप से) हमेशा हमारे लिए एक होते हैं" does not seem to have the same meaning as "we (by default) always have one for us as well"

File: examples/tutorials/hindi/Part 01 - The Basic Tools of Private Deep Learning.ipynb [hunk 8]
```diff
@@ -0,0 +1,965 @@
...
+    "और जैसा कि आप देख सकते हैं ... बॉब (Bob) में अब टेंसर्स नहीं हैं !!! वे हमारी मशीन में वापस चले गए हैं!"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Section 1.2 - Using Tensor Pointers\n",
+    "\n",
+    "तो, बॉब (Bob) से Tensor भेजना और प्राप्त करना बहुत अच्छा है, लेकिन यह शायद ही डीप लर्निंग है! हम दूरस्थ टेनर्स पर Tensor _operations_ का प्रदर्शन करने में सक्षम होना चाहते हैं। सौभाग्य से, Tensor पॉइंटर् (Pointers) स इसे काफी आसान बनाते हैं! आप बस Pointers का उपयोग कर सकते हैं जैसे आप सामान्य Tensors करेंगे!"
```
Comment (Yugandhartripathi) [inline examples/tutorials/hindi/Part 01 - The Basic Tools of Private Deep Learning.ipynb]: टेनर्स to Tensor  
पॉइंटर् (Pointers) स move last letter to left of parenthesis

File: examples/tutorials/hindi/Part 01 - The Basic Tools of Private Deep Learning.ipynb [hunk 9]
```diff
@@ -0,0 +1,965 @@
...
+    "z"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "तथा देखा (And voilà!) \n",
+    "\n",
+    "पर्दे के पीछे, बहुत शक्तिशाली कुछ हुआ। x और y के बजाय स्थानीय रूप से एक computing करने के बजाय, एक कमांड (Commend) को क्रमबद्ध किया गया और बॉब(Bob) को भेजा गया, जिन्होंने गणना करने के लिए, एक Tensor z बनाया, और फिर पॉइंटर(Pointer) को हमें वापस z करने के लिए वापस कर दिया!\n",
```
Comment (Yugandhartripathi) [inline examples/tutorials/hindi/Part 01 - The Basic Tools of Private Deep Learning.ipynb]: Commend to Command  
" एक Tensor z बनाया, और फिर पॉइंटर(Pointer) को हमें वापस z करने के लिए वापस कर दिया!" needs a fix

File: examples/tutorials/hindi/Part 01 - The Basic Tools of Private Deep Learning.ipynb [hunk 10]
```diff
@@ -0,0 +1,965 @@
...
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# बधाई हो!!! - समुदाय में शामिल होने का समय!\n",
+    "\n",
+    "इस नोटबुक ट्यूटोरियल को पूरा करने पर बधाई! यदि आपने इसका आनंद लिया है और एआई और एआई आपूर्ति श्रृंखला (डेटा) के विकेन्द्रीकृत स्वामित्व के संरक्षण की ओर आंदोलन में शामिल होना चाहते हैं, तो आप निम्न तरीकों से ऐसा कर सकते हैं!\n",
+    "\n",
+    "### Pysyft को Github पर लाइक करें!\n",
+    "\n",
+    "हमारे समुदाय की मदद करने का सबसे आसान तरीका सिर्फ GitHub repos अभिनीत है! यह हमारे द्वारा बनाए जा रहे कूल टूल्स के बारे में जागरूकता बढ़ाने में मदद करता है।\n",
```
Comment (Yugandhartripathi) [inline examples/tutorials/hindi/Part 01 - The Basic Tools of Private Deep Learning.ipynb]: Remove सिर्फ that makes it sound restrictive.

File: examples/tutorials/hindi/Part 01 - The Basic Tools of Private Deep Learning.ipynb [hunk 11]
```diff
@@ -0,0 +1,965 @@
...
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# बधाई हो!!! - समुदाय में शामिल होने का समय!\n",
+    "\n",
+    "इस नोटबुक ट्यूटोरियल को पूरा करने पर बधाई! यदि आपने इसका आनंद लिया है और एआई और एआई आपूर्ति श्रृंखला (डेटा) के विकेन्द्रीकृत स्वामित्व के संरक्षण की ओर आंदोलन में शामिल होना चाहते हैं, तो आप निम्न तरीकों से ऐसा कर सकते हैं!\n",
+    "\n",
+    "### Pysyft को Github पर लाइक करें!\n",
```
Comment (Yugandhartripathi) [inline examples/tutorials/hindi/Part 01 - The Basic Tools of Private Deep Learning.ipynb]: Like to star for proper terminology

File: examples/tutorials/hindi/Part 01 - The Basic Tools of Private Deep Learning.ipynb [hunk 12]
```diff
@@ -0,0 +1,965 @@
...
+    "\n",
+    "- [Star PySyft](https://github.com/OpenMined/PySyft)\n",
+    "\n",
+    "### हमारी Slack में शामिल हों!\n",
+    "\n",
+    "नवीनतम प्रगति पर अद्यतित रहने का सबसे अच्छा तरीका हमारे समुदाय में शामिल होना है! आप फॉर्म भरकर ऐसा कर सकते हैं। [http://slack.openmined.org](http://slack.openmined.org)\n",
+    "\n",
+    "### एक कोड प्रोजेक्ट में शामिल हों!\n",
+    "\n",
+    "हमारे समुदाय में योगदान करने का सबसे अच्छा तरीका एक कोड योगदानकर्ता बनना है! किसी भी समय आप PySyft GitHub जारी करने वाले पृष्ठ पर जा सकते हैं और \"प्रोजेक्ट्स\" के लिए फ़िल्टर कर सकते हैं। यह आपको सभी शीर्ष स्तर के टिकट दिखाएगा कि आप किन परियोजनाओं में शामिल हो सकते हैं! यदि आप किसी परियोजना में शामिल नहीं होना चाहते हैं, लेकिन आप थोड़ी सी कोडिंग करना चाहते हैं, तो आप \"अच्छा पहला अंक\" चिह्नित गीथहब मुद्दों की खोज करके अधिक \"वन ऑफ़\" मिनी-प्रोजेक्ट्स की तलाश कर सकते हैं।\n",
```
Comment (Yugandhartripathi) [inline examples/tutorials/hindi/Part 01 - The Basic Tools of Private Deep Learning.ipynb]: Change PySyft GitHub to PySyft GitHub Issues

**Comment by iamtrask** [comment]
Well done!

**Comment by Yugandhartripathi** [comment]
@darkmatter18 Hey, I think there is a problem here I just noticed while reviewing #2811  that you fixed all issues here in commit a6610c0 but then just reverted back to the mistakes in the next one(f9c18d1).

**Comment by darkmatter18** [comment]
@Yugandhartripathi Hey, Thanks for helping. I will solve them in another pull request

**Pull request merged**